products/paas/setup-template.md

```diff
@@ -0,0 +1,107 @@
+# Setup Template
+
+The setup template is a derivation from the [shopware/production](https://github.com/shopware/production) template. It contains build and deployment logic for Shopware PaaS as well as configuration for infrastructure and services. In this chapter we will have a look at these customizations.
+
+Below is a directory overview of the PaaS setup template.
+
+```
+shopware-paas/
+├─ .platform/
+│  ├─ routes.yaml
+│  ├─ services.yaml
+├─ bin/
+├─ config/
+├─ custom/
+├─ files/
+│  ├─ theme-config/
+├─ src/
+├─ .platform.app.yaml
+```
+
+## [.platform.app.yaml](https://github.com/shopware/paas/blob/main/.platform.app.yaml)
+
+This file contains Shopware PaaS specific configuration and can be customized as needed for your individual project.
+
+### name
+
+Is the name of your app. It's used in commands like
+
+```bash
+shopware ssh -A app 'bin/console theme:dump'
+```
+
+Unless there's specific need for it, leave it as `app`.
+
+### type
+
+The base image used for your build process.
+
+### variables
+
+This section contains configuration for environment variables or server settings. General store settings and configuration are set here. In this place you can inject custom environment variables or enable feature flags.
+
+Variables in the `env` section are automatically injected as environment variables.
+
+### hooks
+
+Lifecycle hooks are custom scripts that are called during your build and deploy processes. See more on the [deployment process](./build-deploy#push-main-branch).
+
+#### build hook
```

Comment:
Very important to say:

- In this step you don't have access to all services, only the file system is available
- All files which are not in a `mount` are here only writeable and later steps readonly
- The best practise is to do the most stuff in the `build` as the `deploy` is blocking the web traffic